### DISCUSSION TITLE
Add an "About" page in the main menu

### DISCUSSION THREAD

**Discussion by DeflateAwning** [PaulWoitaschek/Voice#1468] [Ideas]
### What happened?

I had a hard time finding how to get here (to this Github repo) to open this ticket. Adding an "About" page, with a link to this repo and this bug report page, would be very helpful for users of the app. 

Perhaps the "About" page could be accessed from the Settings menu, or maybe the link could just be added to the bottom of the settings page itself.

**Specifically, I'd suggest:**
* The app's version
* Link to the github repo
* License name
* Link to the bug report page

### Version

5.0.2 and 6.0.7

**Comment by PaulWoitaschek** [comment]
Thanks for the suggestion! You can click on the heart in the top right of the settings menu.

**Reply by DeflateAwning** [reply]
You're right, but that page is missing the license and version number. I also didn't notice it (it's pretty well hidden, and not obvious what the heart represents). The info circle ('i' inside a circle) would likely work better.

**Reply by PaulWoitaschek** [reply]
Great feedback!

Why do you think, showing a license is important?

![grafik](https://user-images.githubusercontent.com/743462/187524674-275090db-c142-44e5-bb5e-e825d8196b5f.png)

**Reply by DeflateAwning** [reply]
I think you might legally have to show the license (depending on which one is in use). It also demonstrates that the app is clearly open source, and that feedback/advice will actually be read by a caring person (as opposed to being sorted and filtered by bUsIneSs vAlUe, like with sizable apps). Not essential though, I agree. I like that display of the version!
